### DISCUSSION TITLE
integration and analysis across conditions/timepoints

### DISCUSSION THREAD

**Discussion by aminzia** [satijalab/seurat#4666] [Q&A] [answered]
Dear Seurat Team, 

In many past comments and feedback, you and others have suggested that after using any kind of integration (which basically tries to remove the batch biases), one needs to go back and use original assays (RNA) when looking at markers for DEG analysis, visualization, etc. 

While I understand why you recommend this (to avoid inaccuracies imposed by rotation of different populations of cells by different extents and therefore skewed/distorted relative distances between within-sample cell populations), I still don't understand how you could justify using original (e.g. RNA) assays for e.g. comparing markers in two time-points or between two conditions. If I understand it correctly, the batch effect is still out there in the RNA assays and any comparison of markers between batches is still biased/influenced by the batch effects. So, none of the comparisons in, e.g., dotplots here seem to be accurate (https://satijalab.org/seurat/articles/integration_introduction.html).

Since what I say might not be accurate, I wanted to ask you guys to let us all know what you think? 
Specifically, in your opinion, is comparing one marker between two time-points or two conditions using original RNA assays is influenced by the batch effect or not? if yes, wouldn't it be more accurate to use data after integration to compare markers between time points? 

Thank you so much
Amin

**Comment by timoast** [comment, marked as answer]
We recommend including batch information as a latent variable in the differential expression model (see https://github.com/satijalab/seurat/discussions/4000), rather than running DE on the integrated data. Similarly, we recommend visualizing the original uncorrected data across replicates or batches, rather than the integrated data.
